lib/types/uri-type.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const URI_TYPE = Object.freeze({
+  TRACK: 0,
+  LINE_IN: 1,
+  RADIO: 2
+});
+
```

Comment:
Actually, maybe it's nicer to have human readable values for these enums, so we can expose them directly in the state data. Instead of 0,1,2 define them as "track", "line_in" and "radio". 

It still makes sense to use the enum object though, instead of doing string-comparision in the code base.